stable/aws-vpc-cni/README.md

```diff
@@ -60,6 +62,15 @@ The following table lists the configurable parameters for this chart and their d
 | `init.securityContext`  | Init container Security context                         | `privileged: true`                  |
 | `originalMatchLabels`   | Use the original daemonset matchLabels                  | `false`                             |
 | `nameOverride`          | Override the name of the chart                          | `aws-node`                          |
+| `nodeAgent.image.tag`   | Image tag for Node Agent                                | `v1.0.1-eksbuild.1`                            |
```

Comment:
nit: these new entries could be aligned. Also, node agent image tag for public chart is `v1.0.1`

Reply:
Updated! Thanks